examples/remotereconnection/src/index.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const Prism = require('prismjs');
+const Video = require('twilio-video');
+const getSnippet = require('../../util/getsnippet');
+const getRoomCredentials = require('../../util/getroomcredentials');
+const helpers = require('./helpers');
+const handleLocalParticipantReconnectionUpdates = helpers.handleLocalParticipantReconnectionUpdates;
+const handleRemoteParticipantReconnectionUpdates = helpers.handleRemoteParticipantReconnectionUpdates;
+
+const p1Media = document.getElementById('p1-media');
+const P1simulateReconnection = document.getElementById('p1-simulate-reconnection');
+const P2simulateReconnection = document.getElementById('p2-simulate-reconnection');
+
+// Update UI to indicate remote side room state changes
+const onRoomStateChange = (participant, newState) => {
+  const oldRoomState = document.querySelector(`#${participant} div.current`)
+  if (oldRoomState) {
+    oldRoomState.classList.remove('current');
+  }
+
+  const newRoomState = document.querySelector(`#${participant} div.${newState}`)
+  newRoomState.classList.add('current');
+}
+
+//Get the Tracks of the given Participant.
+function getTracks(participant) {
+  return Array.from(participant.tracks.values()).filter(function(publication) {
+    return publication.track;
+  }).map(function(publication) {
+    return publication.track;
+  });
+}
+
+(async function() {
+  // Load the code snippet.
+  const snippet = await getSnippet('./helpers.js');
+  const pre = document.querySelector('pre.language-javascript');
+  
+  pre.innerHTML = Prism.highlight(snippet, Prism.languages.javascript);
+  
+  // Get the credentials to connect to the Room.
+  const credsP1 = await getRoomCredentials();
+  const credsP2 = await getRoomCredentials();
+  
+  // Create room instance and name for participants to join.
+  const roomP1 = await Video.connect(credsP1.token, {
+    region: 'au1'
+  });
+  
+  // Set room name for participant 2 to join.
+  const roomName = roomP1.name;
+  
+  // Connecting remote participants.
+  const roomP2 = await Video.connect(credsP2.token, {
+    name: roomName,
+    region: 'au1'
+  });
+  
+  // Appends video/audio tracks when each participant is subscribed.
+  roomP1.on('participantConnected', () => {
+    const localUser = roomP1.localParticipant;
+    getTracks(localUser).forEach(track => {
+      p1Media.appendChild(track.attach());
+    })
+  });
```

Comment:
@PikaJoyce You need not wait for the RemoteParticipant to join to attach the LocalParticipant's Tracks:
```suggestion
  getTracks(roomP1.localParticipant).forEach(track => {
    p1Media.appendChild(track.attach());
  });
```

Reply:
@PikaJoyce Also, can you attach the Tracks of the RemoteParticipant to the remote view as well?
```js
roomP1.on('trackSubscribed', track => {
  p2Media.appendChild(track.attach());
});
```
And just to be sure that you get this event, please move the above two code snippets before `roomP2` is created.